config/betterquesting/resources/supersymmetry/lang/en_us.lang

```diff
@@ -1538,19 +1538,19 @@ susy.quest.db.742.desc=§lGo to the Transportation category of the questbook for
 susy.quest.db.746.title=Driving your Trains
-susy.quest.db.746.desc=Right clicking an Engine makes you go inside, allowing you to drive the train.%n%nBy default, you use the keypad to control the train. You can change this in Options > Controls. If you want to keep it that way, you will need NumLock enabled for the keypad to work.%n%nThe controls, as well as further instructions, can be seen here: §9https://github.com/TeamOpenIndustry/ImmersiveRailroading/wiki/Driving-Diesel-Locomotives
+susy.quest.db.746.desc=right-clicking an Engine makes you go inside, allowing you to drive the train.%n%nBy default, you use the keypad to control the train. You can change this in Options > Controls. If you want to keep it that way, you will need NumLock enabled for the keypad to work.%n%nThe controls, as well as further instructions, can be seen here: §9https://github.com/TeamOpenIndustry/ImmersiveRailroading/wiki/Driving-Diesel-Locomotives
 susy.quest.db.747.title=Fueling your Trains
-susy.quest.db.747.desc=Only Engines need to be fueled before they can work. They can be fueled by two ways:%n%n1. Right clicking the Engine to enter, and then right clicking again to open the fuel GUI, which you can use to add fuel or remove fuel%n%n2. Having the Engine pass over a Fluid Loader, which can automatically add fuel to the Engine%n%nThe following fuels can be used to fuel the Engine:%n- Diesel%n- Gasoline%n- Naphtha%n- Fuel Oil%n- Kerosene%n- Biodiesel%n- Cetane-Boosted Diesel%n- Midgrade Gasoline%n- Premium Gasoline%n- Supreme Gasoline
+susy.quest.db.747.desc=Only Engines need to be fueled before they can work. They can be fueled by two ways:%n%n1. right-clicking the Engine to enter, and then right-clicking again to open the fuel GUI, which you can use to add fuel or remove fuel%n%n2. Having the Engine pass over a Fluid Loader, which can automatically add fuel to the Engine%n%nThe following fuels can be used to fuel the Engine:%n- Diesel%n- Gasoline%n- Naphtha%n- Fuel Oil%n- Kerosene%n- Biodiesel%n- Cetane-Boosted Diesel%n- Midgrade Gasoline%n- Premium Gasoline%n- Supreme Gasoline
 susy.quest.db.748.title=Cargo Trains
-susy.quest.db.748.desc=There are also Monorail Tanks, Monorail Freights, and Monorail Passenger Chaircars. They are made similarly to the Monorail Engine, and can be connected to the Engine by simply having them bump into each other on a track.%n%nThe Monorail Tank can store 2,500 buckets, or 2,500,000 L of any liquid. They can be interacted with using Fluid Loaders/Unloaders, or by Shift+Right Clicking the Tank to open a GUI for adding or removing fluids.%n%nThe Monorail Freight can store 6,400 items. They can be interacted with using Item Loaders/Unloaders, or by Shift+Right Clicking the Tank to open a GUI for adding or removing items.%n%nRight clicking a Monorail Passenger Chaircar makes you go inside. You can move around.
```

Comment:
Again, why not turn "shift" into lowercase here?

Reply:
Wait, actually, the third "right click" here is lowercase at the start of a sentence!

Reply:
fixed